state/protocol/badger/mutator.go

```diff
@@ -599,17 +525,10 @@ func (m *FollowerState) checkBlockAlreadyProcessed(blockID flow.Identifier) (boo
 // `block` is orphaned or already finalized.
 //
 // Expected errors during normal operations:
-<<<<<<< HEAD
 //   - [state.OutdatedExtensionError] if the candidate block is orphaned or finalized
 func (m *ParticipantState) checkOutdatedExtension(block *flow.Header) error {
 	var latestFinalizedHeight uint64
 	err := operation.RetrieveFinalizedHeight(m.db.Reader(), &latestFinalizedHeight)
-=======
-//   - state.OutdatedExtensionError if the candidate block is outdated (e.g. orphaned)
-func (m *ParticipantState) checkOutdatedExtension(header flow.HeaderBody) error {
```

Comment:
I think we should preserve the `flow.HeaderBody` variant, there is no need to pass the `*flow.Header` to this function.